From c28f41d7bb080caaf04eb7779ebcbe87bba17333 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Mon, 9 Feb 2015 12:25:32 -0500
Subject: [PATCH] KVM: ia64: drop kvm.h from installed user headers

The header was deleted, so stop trying to install it.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 arch/ia64/include/uapi/asm/Kbuild | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/ia64/include/uapi/asm/Kbuild b/arch/ia64/include/uapi/asm/Kbuild
index 1b3f5eb..891002b 100644
--- a/arch/ia64/include/uapi/asm/Kbuild
+++ b/arch/ia64/include/uapi/asm/Kbuild
@@ -18,7 +18,6 @@ header-y += intrinsics.h
 header-y += ioctl.h
 header-y += ioctls.h
 header-y += ipcbuf.h
-header-y += kvm.h
 header-y += kvm_para.h
 header-y += mman.h
 header-y += msgbuf.h
-- 
2.2.1

